src/graphql/search/filters/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Context } from '../../config'
+import getCmsFilters from '../cms/filters'
+import { DCAT_ENDPOINTS } from '../datasets/config'
+import getDatasetsFilters from '../datasets/filters'
+import { combineFilters } from './utils'
+
+export default async (_: any, {}, { loaders }: Context) => {
```

Comment:
The return type of this method is now implicitly `any` since the error returned is of type `any`.

Reply:
Given the answer above, do I still need to write `Promise<Array<Filter> | Error>` as return type? Or will it be known?

Reply:
Adding `return e as Error` should make the return type `Promise<Array<Filter> | Error>` automatically.